FastQC/0.11.8/FastQC.nf

```diff
@@ -1,22 +1,18 @@
-params.fastqc
 
 process FastQC {
-    tag "${sample}_fastqc"
-    publishDir "$params.outdir/$sample/FastQC", mode: 'copy'
-    cpus 1
-    penv 'threaded'
-    memory '1 GB'
-    time '1h'
+    tag {"FastQC ${sample_id} - ${rg_id}"}
+    publishDir "$params.out_dir/$sample_id/FastQC", mode: 'copy'
 
     input:
-    set val(sample), file(fastq: "*")
+    tuple sample_id, rg_id, file(fastq: "*")
```

Comment:
What is the difference between a tuple and set?

Reply:
tuple replaces set in latest version of Nextflow